pkg/processor/sourceprocessor/README.md

```diff
@@ -84,3 +84,36 @@ prefixed with the value defined in `keys.annotation_prefix` config option.
 This can be achieved with the [Kubernetes processor](../k8sprocessor).
 
 For example, if a resource has the `k8s.pod.annotation.sumologic.com/exclude` attribute, the resource will be dropped.
+
+### Container-level pod annotations
+
+To make it possible to set different metadata on logs from different containers inside a pod,
+it is possible to set pod annotations that are container-specific.
+Container-level annotations take precendence over other forms of setting the source category.
```

Comment:
can we bold it?

Reply:
I'm not a fan of bolding half of the docs ;)
So I tried the make the most relevant points more visible moving them into a list near the top of the docs. How about this? https://github.com/SumoLogic/sumologic-otel-collector/pull/260/commits/0cff98d39e82c7df535f91275dc91c0439ce30b8

Reply:
It's better. It's little but harder to miss it while reading